CMakeLists.txt

```diff
@@ -45,12 +51,15 @@ endif()
 #       Then, this needs to read `add_library(boost_json INTERFACE)`
 #       and related settings need to be INTERFACE-ed as well.
 add_library(boost_json ${BOOST_JSON_HEADERS} ${BOOST_JSON_SOURCES})
+set_target_properties(boost_json PROPERTIES EXPORT_NAME json)
 add_library(Boost::json ALIAS boost_json)
-
 target_compile_features(boost_json PUBLIC cxx_constexpr)
 
 # TODO: For Boost superproject, this may need to be INTERFACE setting
-target_include_directories(boost_json PUBLIC include)
+# Installed targets have different include requirements
+target_include_directories(boost_json PUBLIC
+        "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+        "$<INSTALL_INTERFACE:include>")
```

Comment:
It's more idiomatic to use `include(GNUInstallDirs)` and then `${CMAKE_INSTALL_INCLUDEDIR}` here.